pkg/kubernetes/deployment/appsv1/v1alpha1/kubernetes.go

```diff
@@ -168,6 +231,31 @@ func (k *Kubeclient) GetRaw(name string) ([]byte, error) {
 	return json.Marshal(d)
 }
 
+// Delete deletes a deployment instance from the
+// kubernetes cluster
+func (k *Kubeclient) Delete(name string, deleteOpts *metav1.DeleteOptions) error {
```

Comment:
Is it better to have opts or options rather than having deleteOpts since it is already known that it is Delete() method for deployment.

Reply:
Done